Source/common/SNTConfigurator.m

```diff
@@ -1093,6 +1100,11 @@ - (BOOL)fcmEnabled {
   return (self.fcmProject.length && self.fcmEntity.length && self.fcmAPIKey.length);
 }
 
+- (BOOL)enableAPNS {
+  NSNumber *number = self.configState[kEnableAPNS];
```

Comment:
We should make it possible to enable this from the sync server (not _necessarily_ in this PR)

Reply:
Added a TODO for now.